portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -79,7 +64,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Gson gson = new Gson();
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(tasks));
-    response.sendRedirect("/login");
   }
 
   private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
This method can have some Java doc. 
Describe the action the method performs. What the method returns is sometimes worth incorporating into the summary fragment (…, and returns …), but may also just be given later in the @return clause.

Reply:
It would be helpful to have some comments for doGet and doPost.

Reply:
Done! Let me know what you think of the documentation, sometimes writing good concise comments can be a bit difficult for me.

Reply:
LGTM